apps/fishing-map-e2e/src/e2e/map/map.cy.ts

```diff
@@ -0,0 +1,49 @@
+import { API_URL_4WINGS_TILES, API_URL_VESSELS, URL_FULL_DATA_AREA } from '../../constants/urls'
+import {
+  afterTestSkip,
+  beforeTestSkip,
+  disablePopups,
+  getTimeline,
+  waitForMapLoadTiles,
+} from '../../support/app.po'
+
+describe('map', () => {
+  before(() => beforeTestSkip())
+  afterEach(afterTestSkip)
+
+  it('should pin a vessel from a cell in the map', () => {
+    disablePopups()
+    cy.visit(URL_FULL_DATA_AREA)
+    waitForMapLoadTiles(3000) // Give 3 seconds more to paint data on the map
+    cy.intercept(API_URL_VESSELS).as('searchForVessels')
+    cy.get('.maplibregl-canvas').click('center')
+    cy.get('.scrollContainer').scrollTo('center', { easing: 'linear', duration: 2000 })
+    cy.wait('@searchForVessels', { requestTimeout: 10000 })
+    cy.getByClass('VesselsTable_vesselsTable').find('button').first().click()
+    cy.getByClass('VesselsTable_vesselsTable')
+      .find('tbody')
+      .find('tr')
+      .first()
+      .find('td')
+      .eq(1)
+      .then((vessel) => {
+        const textValue = vessel.text()
+        cy.wrap(textValue).as('firstVessel')
+      })
```

Comment:
Using class, id or tag selectors as well as navigating the DOM structure in our e2e tests is an [anti-pattern](https://docs.cypress.io/guides/references/best-practices#Selecting-Elements). They are subject to change and will break our tests sooner than later.